### PR TITLE
Tidy up

### LINKED ISSUE #38: Add ngrok installation
https://ngrok.com/

Install with: [`brew install ngrok`](http://brewformulas.org/ngrok) (`brew cask install ngrok`)

### LINKED ISSUE #41: Add bash if statement to speed up nvm on new terminal instance
From comment here: https://github.com/samhstn/My-Config/commit/fa94daa02216dbd83f3dcfb6c8ca447ff46a97be#r29606326 from @ivanmauricio.

This `if` statement should be added back in

### LINKED ISSUE #42: Install script not working
The commmands from [here](https://github.com/samhstn/My-Config#config-installation) don't work. The branch it is referencing no longer exists, should reference the `master` branch instead

### LINKED ISSUE #43: Gitter link broken
The gitter link in `README.md` is broken, linking to https://gitter.im/shouston3, should be https://gitter.im/samhstn

### PR DESCRIPTION
**Pull request by samhstn**
- [x] Remove specific osx version, this will always be changing
- [x] Add command to improve `$ gb` command (https://stackoverflow.com/a/48370253)
- [x] Fix gitter badge broken link (#43)
- [x] Remove `sass` - don't use it, only use `node-sass`
- [x] Add `aws` cli tool installation
- [x] Remove `script.sh` and add install commands inline in `README` instead - to promote a better understanding of what is happening.

Fixes #38 
Fixes #41 
Fixes #42 
Fixes #43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +20,7 @@ This is my OSX setup for Sierra 10.12. I like a simple setup.
 
 + [**Git**](https://github.com/shouston3/My-Config/tree/master#git)
 
-+ [**Sass**](https://github.com/shouston3/My-Config/tree/master#sass)
++ [**AWS**](https://github.com/shouston3/My-Config/tree/master#aws)
```
Comment (nelsonic) [inline README.md:23]: glad to see this switch. 🔧

File: script.sh [hunk 2]
```diff
@@ -1,45 +0,0 @@
-echo "Installing shouston3's setup..."
```
Comment (nelsonic) [inline script.sh:1]: was this script difficult to maintain or just unused?

**Review by nelsonic — APPROVED** [review]
@samhstn updates look great to me. 👀 👍 :shipit:

File: script.sh (hunk 2 above)
Comment (samhstn) [reply script.sh:1]: I'm just not a fan of telling someone to run some script (which they might not check).
I thought forcing the user to see each command being run would be better - I also found myself opening the script to run each line instead.

Comment (nelsonic) [reply script.sh:1]: @samhstn makes sense. 👍

File: sams-config.plugin.zsh [hunk 3]
```diff
@@ -2,37 +2,29 @@ alias desk='cd ~/Desktop && ls'
...
-alias mc='cd ~/proj/my-config'
+alias za='mvim -v ~/.oh-my-zsh/plugins/sams-config/sams-config.plugin.zsh'
+alias ze='mvim -v ~/.zsh_envs'
 alias m='mvim -v'
+alias ms='mix phx.server'
+alias mpr='mix phx.routes'
+alias python=python3
+alias pip=pip3
+alias dj='python manage.py'
+alias ggpuhs=ggpush
```
Comment (RobStallion) [inline sams-config.plugin.zsh:24]: I assume that this is just to catch a typo you often make?

**Review by RobStallion — APPROVED** [review]
Looks good to me 👍 

Really like the swapping of the script for the inline commands 👍

File: sams-config.plugin.zsh (hunk 3 above)
Comment (samhstn) [reply sams-config.plugin.zsh:24]: exactly 👍

**Pull request merged**